iphone/Maps/UI/Storyboard/Main.storyboard

```diff
@@ -18,7 +18,7 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ixC-IZ-Pvs" userLabel="CarPlayPlaceholderView" customClass="CarplayPlaceholderView" customModule="Organic_Maps" customModuleProvider="target">
-                                <rect key="frame" x="59" y="0.0" width="814" height="409"/>
+                                <rect key="frame" x="0.0" y="0.0" width="932" height="430"/>
```

Comment:
This one can be removed too.

Reply:
I think it is needed to avoid warning about misplaced views.
I changed constraints from Safe Area to screen edges to fill CarPlay placeholder view background to all screen.

<img width="372" alt="image" src="https://github.com/organicmaps/organicmaps/assets/1800899/ac85a724-ce41-4b38-8273-1a44c53881b9">

Reply:
I like removing warnings :) ! Thanks!